python/ray/rllib/eval.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+
```

Comment:
Could you do the import __future__ stuff from train.py as well here?

Reply:
Should be fixed in the latest commit